docs/guide/rest-apis.md

```diff
@@ -297,25 +297,33 @@ Returns an individual page as a JSON object:
 
-### `PATCH /api/v1/@apostrophecms/page/:docId`
+### `PATCH /api/v1/@apostrophecms/page/:id`
 
 **Authentication required.** Identical to PATCHing a piece, except that you may include `_position` and `_targetId` if you wish to change the position in the tree. To easily move a page into the trash, set `_targetId` to `_trash` and `_position` to `lastChild`. You may similarly move pages out of the trash by moving them to a position relative to a page that is not in the trash.
 
-### `DELETE /api/v1/@apostrophecms/page:docId`
+### `DELETE /api/v1/@apostrophecms/page/:id`
 
 **NOT supported.** Instead set `_targetId` to `_trash` and `_position` to `lastChild`. You may similarly move pages out of the trash by moving them to a position relative to a page that is not in the trash.
 
-## PATCHing a document of any type
+## Locale and mode in API calls
 
-**Authentication required.** As a convenience, you may also `PATCH` a document without specifying the exact module that should handle it:
+Once published, most Apostrophe document types have both a draft and a published copy in the database. If you are using multiple locales in your project, there may be documents for each locale as well. The `apos-mode` and `apos-locale` query parameters help support using REST API routes to manage those versions.
 
-### `PATCH /api/v1/@apostrophecms/doc/:docId`
+Passing `draft` or `published` values on `apos-mode` and your locale names on the `apos-locale` parameter tell Apostrophe to execute that request for those versions of the piece or page you are working with. For example, `GET /api/v1/product?apos-mode=draft&apos-locale=fr` will get all the draft and `fr` (French) locale versions of the product documents. These can be used independently, so you can only use `apos-mode` and Apostrophe will use the default locale value. Or only use `apos-locale` and Apostrophe will use `published` for the mode.
 
-In this case the request is routed to the appropriate module. Apostrophe uses this route for in-context editing.
+### Locale and mode in single-document requests
+
+**The exception to those assumed values are with `GET ONE`, `PATCH`, and other requests that include a document `_id` value.** Document IDs include mode and locale information already. `ckgsj5in400d5xi4lb6fu29rh:fr:draft` ends with `:fr:draft`, indicating that it is the `fr` locale and `draft` version of a particular piece or page.
+
+You may still use the `apos-mode` and `apos-locale` parameters with these requests related to existing documents. If the values for those parameters match the document you are requesting, the request will treat it as if the parameters weren't there. If the values differ from the `_id` value, and a version of the document exists for the mode and locale values, the request will apply to the version matching the parameter values.
```

Comment:
I would just say: "However, if `apos-mode` and/or `apos-locale` query parameters are present, they always take precedence. This makes it easy to request an alternate version of a document whose `_id` you already have."

To say more can lead to inaccuracies. For instance, it is not true that it will only do this if the other version exists. It will definitely do it (and you will get a 404, if appropriate, or create a new document, etc).